### PR TITLE
Bpf120 doc sidebar

### PR DESCRIPTION
**Pull request by bpf120**
Add link to new Dolt test apps page on sidebar

### CODE REVIEW

File: content/reference/sql/sql-support/miscellaneous.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ Dolt supports the following join hints:
 | MERGE_JOIN(<t1>,<t2>)    | ✅         | Use MERGE strategy joining two tables.                                                          |
 | HASH_JOIN(<t1>,<t2>)     | ✅         | Use HASH strategy joining two tables.                                                           |
 | INNER_JOIN(<t1>,<t2>)    | ✅         | Use INNER strategy joining two tables.                                                          |
-| SEMI_JOIN(<t1>,<t2>)     | ❌         | Use SEMI strategy joining two tables.                                                           |
```
Comment (timsehn) [inline content/reference/sql/sql-support/miscellaneous.md:39]: Why are these here?

Comment (bpf120) [reply content/reference/sql/sql-support/miscellaneous.md:39]: no idea. let me try add my summary change to the other pr
